doc/source/user/cleanup-k8s.rst

```diff
@@ -0,0 +1,19 @@
+================================
+Cleanup k8s of socok8s artifacts
+================================
+
+If you want to simply cleanup your kubernetes environment of all the
+traces of a socok8s deployment, run:
+
+.. note ::
+
+   This script is in work in progress.
```

Comment:
I don't think this note is useful, after all it applies for whole repository...

Reply:
Fair.